### PR TITLE
feat: add proper support for getFieldTypes

### PR DESCRIPTION
**Pull request by Deepak-Kharah**
The file is ts ignored as the typescript is not properly provide the types.

fixes: https://contentstack.atlassian.net/browse/VC-179

### CODE REVIEW

File: src/liveEditor/utils/getFieldType.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        fieldSchema?.field_metadata?.markdown
+    ) {
+        return "markdown_rte";
+    }
+    if (fieldSchema.enum) {
+        return "select";
+    }
+    if (fieldSchema.data_type === "text") {
+        return "singleline";
+    }
```
Comment (hiteshshetty-dev) [inline src/liveEditor/utils/getFieldType.ts:28]: We will need to add type for other fields as well.

**Review by hiteshshetty-dev — APPROVED** [review]
Changes looks good to me!!

File: src/liveEditor/utils/getFieldType.ts (hunk 1 above)
Comment (Deepak-Kharah) [reply src/liveEditor/utils/getFieldType.ts:41]: I have added new types, please review

Comment (Deepak-Kharah) [reply src/liveEditor/utils/getFieldType.ts:45]: Yes

**Review by hiteshshetty-dev — APPROVED** [review]
Looks good to me!!

**Pull request merged**